lib/src/feature/calendar/widgets/date_widget.dart

```diff
@@ -91,11 +91,12 @@ class DateWidget extends StatelessWidget {
   }
 
   void _navigateToPreviousWorkoutScreen(context, Workout? workout) {
-    if (workout != null)
+    if (workout != null) {
```

Comment:
Maybe add guard clause here instead of brackets so:

```
if (workout == null) return;
// navigate
```

Reply:
done